controllers/openstackdeploy_controller.go

```diff
@@ -106,6 +107,12 @@ func (r *OpenStackDeployReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		)
 	}
 
+	uuid, _ := exec.Command("uuidgen").Output()
```

Comment:
https://pkg.go.dev/github.com/google/uuid

Reply:
stackoverflow[1] recommends that for some reason. I can change it if that helps.

[1]https://stackoverflow.com/questions/15130321/is-there-a-method-to-generate-a-uuid-with-go-language

Reply:
Already using https://pkg.go.dev/github.com/google/uuid in pkg/common/password.go